support/build/config-submenu.sh

```diff
@@ -58,7 +61,16 @@ while getopts :hqt:l:r:c:o: OPT; do
 		IFS=':'
 		for P in ${OPTARG}; do
 			IFS=$IFS_ORIG
-			ARG_PATHS+=("${P}")
+			[ ! -z "${P}" ] && ARG_PATHS+=("${P}")
+			IFS=$IFS_NL
```

Comment:
Is `shellcheck` used on unikraft helper scripts?

For example here it suggests to use `-n` instead of `! -z`

Reply:
If we want to enforce it we can add a new workflow that runs it and outputs stuff on all shell files (as reported by `file`)

Reply:
Do you want me to change it? Effectively it is the same.

Reply:
I'm indifferent, this is just an example.

My question was more: should we enforce style checks on shell scripts in the upcoming future? I know you're the guy usually editing those so it would be good you know in advance/decide if we want to do this